Makefile.am

```diff
@@ -136,6 +136,10 @@ shellcheck:
 		echo "skipping shellcheck because shellcheck is not installed"; \
 	fi
 
+PHONY += checkabi
+checkabi: lib
+	$(MAKE) -C lib checkabi
```

Comment:
As a convenience it would be nice to have a `make updateabi` target we could run to ensure we always update the ABI files in a consistent way.  Or a script under `./scripts/`.  This would also let us automatically add suppressions (`abidw --suppressions`)  if we determine any are needed.  Exactly what options did you use to generate these .abi files, I don't seem to be able to reproduce them identically.

It might also be nice to include the SOVERSION number in the abi file.  For example, `libnvpair-3.0.0.abi`

Reply:
That's an especially fair comment.  I'll add that in the next revision.